bindgen/callbacks.rs

```diff
@@ -135,6 +135,15 @@ pub trait ParseCallbacks: fmt::Debug {
     fn process_comment(&self, _comment: &str) -> Option<String> {
         None
     }
+
+    /// Allows renaming the name of a field, replacing `_name`.
```

Comment:
Could you specify what `parent_name` is? Assuming it's the struct type, something like `type_name` may be better, since usually parent/child refer to nested types or modules (rather than fields).